cmd/adsysd/integration_tests/adsysctl_policy_test.go

```diff
@@ -97,7 +97,7 @@ func TestPolicyApplied(t *testing.T) {
 		wantErr bool
 	}{
 		"Current user applied gpos": {},
-		"Other user applied gpos":   {args: []string{"UserIntegrationTest@example.com"}, userGPORules: "UserIntegrationTest@example.com"},
+		"Other user applied gpos":   {args: []string{"UserIntegrationTest@example.com"}, userGPORules: "userintegrationtest@example.com"},
```

Comment:
I would keep that test with both args lowercase.

Then, add another test with the user argument not capitalized.

Reply:
Ok, the confusion comes from this comment which predated our quick HO :) I see now, sorry about that :p